### PR TITLE
Updated some sigs

### PR DESCRIPTION
**Pull request by Intoprelised**

### CODE REVIEW

**Comment by Thekingpenguins** [comment]
ys

**Comment by Synx173** [comment]
ok

File: SDK/CBlockLegacy.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@ void C_BlockLegacy::liquidGetFlow(vec3_t* flowOut, C_BlockSource* reg, const vec
 	static liquid_getFlow_t liquid_getDepth = nullptr;
 	if(liquid_getDepth == nullptr){
 		if(g_Data.getVersion() == GAMEVERSION::g_1_16_0)
-			liquid_getDepth = reinterpret_cast<liquid_getFlow_t>(FindSignature("48 8B C4 55 56 57 41 54 41 55 41 56 41 57 48 8D A8 ? ? ? ? 48 81 EC ? ? ? ? 48 C7 45 ? ? ? ? ? 48 89 58 ? 0F 29 70 ? 0F 29 78 ? 44 0F 29 40 ? 48 8B 05 ? ? ? ? 48 33 C4 48 89 45 ? 49 8B F1 4C 89 4C 24"));
+			liquid_getDepth = reinterpret_cast<liquid_getFlow_t>(FindSignature("E8 ? ? ? ? 48 8B 4C 24 ? F3 0F 10 01"));
```
Comment (Synx173) [inline SDK/CBlockLegacy.cpp:33]: This isn't the start of the function, it's another call inside of a function

File: SDK/CClientInstanceScreenModel.h [hunk 2]
```diff
@@ -6,7 +6,7 @@ class C_ClientInstanceScreenModel {
 public:
 	void sendChatMessage(char* message) {
 		using sendChatMessage = void(__fastcall*)(void*, char*);
-		static sendChatMessage sendChatMessageFunc = reinterpret_cast<sendChatMessage>(FindSignature("56 57 41 54 41 55 41 56 41 57 48 8D AC 24 ?? ?? ?? ?? 48 81 EC ?? ?? ?? ?? 48 C7 45 D8 ?? ?? ?? ?? 48 89 9C 24 ?? ?? ?? ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 85 ?? ?? ?? ?? 48 89 54 24 ??") - 2);
+		static sendChatMessage sendChatMessageFunc = reinterpret_cast<sendChatMessage>(FindSignature("48 89 5C 24 18 55 56 57 41 54 41 55 41 56 41 57 48 8D AC 24 E0 F6 FF FF 48 81 EC 20 0A 00 00 48 8B 05 4A 30 DE 00 48 33 C4 48 89 85 10 09 00 00 45 0F B6 E0 44") - 2);
```
Comment (Synx173) [inline SDK/CClientInstanceScreenModel.h:9]: Apparently doesn't work in 1.17.10

Comment (Intoprelised) [reply SDK/CClientInstanceScreenModel.h:9]: i did this on 1.17.2

Comment (Thekingpenguins) [reply SDK/CClientInstanceScreenModel.h:9]: then update the sigs for 1.17.10

Comment (Synx173) [reply SDK/CClientInstanceScreenModel.h:9]: Also the sig has no wildcards, that's why it broke in 1.17.10

File: SDK/CBlockLegacy.cpp (hunk 1 above)
Comment (Intoprelised) [reply SDK/CBlockLegacy.cpp:33]: whats ur discord

Comment (Synx173) [reply SDK/CBlockLegacy.cpp:33]: Its in my profile

> whats ur discord

**Review by qewrwer23qf — APPROVED** [review]

**Comment by Intoprelised** [comment]
ima update the sigs for 1.17.10 instead

**Pull request closed without merging**